iroh-net/src/net/interfaces.rs

```diff
@@ -70,6 +70,38 @@ impl Interface {
             .map(IpNet::V4)
             .chain(self.iface.ipv6.iter().cloned().map(IpNet::V6))
     }
+
+    #[cfg(test)]
+    pub(crate) fn fake() -> Self {
```

Comment:
maybe add a doc comment for why this exists/what it is used for

Reply:
[2ad536b](https://github.com/n0-computer/iroh/pull/1266/commits/2ad536bffb87e34c9bd958d89d6b94fabec289aa)